tests/test_import_csv.py

```diff
@@ -0,0 +1,125 @@
+import mock
+
+from import_addressbase.import_csv import make_es_actions, make_es_mappings
+from import_addressbase.record_types import DPA
```

Comment:
As it's already been like this, it's ok for now, but it would be best to extend DPA to a descriptive name